src/cfnlint/customRules/Operators.py

```diff
@@ -2,17 +2,42 @@
 Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 SPDX-License-Identifier: MIT-0
 """
+# pylint: disable=W0622
+from cfnlint.rules import Match
 
 
-def equalsOp(rule, propertyList):
+def equalsOp(template, rule, propertyList):
     """ Process EQUALS operators """
+
     if not propertyList:
-        return 'EQUALS'  # Resource type not found
+        return Match(
+            1, 1,
+            1, 1,
+            template.filename, CustomRule('E9200'),
+            rule.operator, None)  # Resource type not found
+
     for prop in propertyList:
         actualValue = getProperty(prop, rule)
         if actualValue.strip() != str(rule.value).strip():
-            return 'Not Equal as ' + actualValue + ' does not equal ' + rule.value
-    return 'EQUALS'
+            path = prop['Path']
+            path = path + rule.prop.split('.')
+            linenumbers = template.get_location_yaml(template.template, path)
+            if linenumbers:
+                return Match(
+                    linenumbers[0] + 1, linenumbers[1] + 1,
+                    linenumbers[2] + 1, linenumbers[3] + 1,
+                    template.filename, CustomRule('E9201'),
+                    'Not Equal as ' + actualValue + ' does not equal ' + rule.value, None)
```

Comment:
think we need to be appending each individual match to a list of Matches to return at the end

Reply:
Multi-Match support per custom rule has been added